mkdocs_jupyter/plugin.py

```diff
@@ -53,13 +54,34 @@ def on_pre_page(self, page, config, files):
             kernel_name = self.config["kernel_name"]
 
             def new_render(self, config, files):
-                body = convert.nb2html(page.file.abs_src_path, execute=exec_nb, kernel_name=kernel_name)
+                body = convert.nb2html(
+                    page.file.abs_src_path, execute=exec_nb, kernel_name=kernel_name
+                )
                 self.content = body
                 self.toc = get_nb_toc(page.file.abs_src_path)
 
             # replace render with new_render for this object only
             page.render = new_render.__get__(page, Page)
 
+            # Add metadata for template
+            self._set_nb_url(page)
+        elif str(page.file.abs_src_path).endswith(".py"):
```

Comment:
not sure what all of the extra stuff here does, so i just copied it. it can probably be simplified